test/unit_tests/aievec_tests/bf16_exp_lut/exp_lut.h

```diff
@@ -0,0 +1,246 @@
+#ifndef __EXP_LUT_H__
+#define __EXP_LUT_H__
+#include "adf.h"
+#include "aie_api/aie.hpp"
```

Comment:
We can relocate this to aie/aie_runtime_lib/AIE2.  If you're not familiar, I can show you how to update the cmake files.